doc/reference/src/main/asciidoc/configuration.asciidoc

```diff
@@ -93,43 +90,87 @@ You can configure the CDI TCK to just run tests related to the CDI Lite specific
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
-                <excludedGroups>cdi-full,integration,javaee-full,se</excludedGroups>
+              <excludedGroups>cdi-full,se</excludedGroups>
+              <dependenciesToScan>
+                <dependency>jakarta.enterprise:cdi-tck-core-impl</dependency>
+              </dependenciesToScan>
             </configuration>
        </plugin>
 ----
 
 === Using the CDI TCK with the Jakarta EE Web Profile
 
-You can configure the CDI TCK to just run tests appropriate to the Jakarta EE Web Profile by excluding TestNG group _javaee-full_, e.g. via a maven-surefire-plugin configuration like:
+You can configure the CDI TCK to just run tests appropriate to the Jakarta EE Web Profile by excluding TestNG group _javaee-full_,_se_ e.g. via a maven-surefire-plugin configuration like:
 
 [source.XML, xml]
 ----
       <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <excludedGroups>javaee-full,se</excludedGroups>
+              <dependenciesToScan>
+                <dependency>jakarta.enterprise:cdi-tck-core-impl</dependency>
+                <dependency>jakarta.enterprise:cdi-tck-web-impl</dependency>
+              </dependenciesToScan>
+            </configuration>
+       </plugin>
+----
+
+=== Using the CDI TCK with the Jakarta EE Full Platform
+
+You can configure the CDI TCK to just run tests appropriate to the Jakarta EE Full Platform by excluding TestNG group _se_ e.g. via a maven-surefire-plugin configuration like:
+
+[source.XML, xml]
+----
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
-                <excludedGroups>javaee-full</excludedGroups>
+              <excludedGroups>se</excludedGroups>
+              <dependenciesToScan>
+                <dependency>jakarta.enterprise:cdi-tck-core-impl</dependency>
+                <dependency>jakarta.enterprise:cdi-tck-web-impl</dependency>
+              </dependenciesToScan>
             </configuration>
        </plugin>
 ----
 
+=== Using the CDI TCK with the Java SE
+
+You can configure the CDI TCK to just run tests appropriate to the Java SE runtime by including the TestNG group _se_ and _arquillian_, e.g. via a maven-surefire-plugin configuration like:
+
+[source.XML, xml]
+----
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <groups>se,arquillian</groups>
+              <dependenciesToScan>
+                <dependency>jakarta.enterprise:cdi-tck-core-impl</dependency>
+            </configuration>
```

Comment:
This snippet misses one line to close the element:
</dependenciesToScan>
before the line 152
</configuration>